osm/src/main/groovy/org/orbisgis/osm/OSMTools.groovy

```diff
@@ -8,38 +8,53 @@ import org.slf4j.LoggerFactory
 import static java.nio.charset.StandardCharsets.UTF_8
 
 /**
- * Main script to access to all processes used to extract, transform and save OSM data as GIS layers
+ * Main script to access to all processes used to extract, transform and save OSM data as GIS layers.
+ *
  * @author Erwan Bocher CNRS LAB-STICC
  * @author Elisabeth Le Saux UBS LAB-STICC
+ * @author Sylvain PALOMINOS (UBS LAB-STICC 2019)
  */
 abstract class OSMTools extends GroovyProcessFactory {
 
+    /** Url of the status of the Overpass server */
     private static final OVERPASS_STATUS_URL = "http://overpass-api.de/api/status"
+    /** {@link Logger} */
     public static Logger logger = LoggerFactory.getLogger(OSMTools.class)
 
-    //Process scripts
+    /* *********************** */
+    /*     Process scripts     */
+    /* *********************** */
+    /** {@link Loader} script with its {@link Process}. */
     public static Loader = new Loader()
+    /** {@link Transform} script with its {@link Process}. */
     public static Transform = new Transform()
+    /** {@link Utilities} script with its {@link Process}. */
     public static Utilities = new Utilities()
 
-    //Utility methods
-    static def getUuid() { UUID.randomUUID().toString().replaceAll("-", "_") }
-    static def uuid = { UUID.randomUUID().toString().replaceAll("-", "_") }
```

Comment:
Why changing this name ?

Reply:
We had two declaration of the 'uuid' method:
 - `def uuid = {...}` was for a normal calling (outside a String) by using `uuid()`
 - `def getUuid() {...}` was for calling inside a String by using `"$uuid`

Now with the propertyMissing method, there is only one implementation which works everywheren and which is more compliant with the groovy syntax/mecanism

Reply:
Fix in a better way with just `static def getUuid(){...}`.